src/bundle/Resources/views/themes/admin/content/modal/location_trash.html.twig

```diff
@@ -15,6 +15,11 @@
     {% endblock %}
     {% block body_content %}
         {{ 'trash.modal.send_to_trash.message'|trans|desc('Are you sure you want to send this Content item to Trash?') }}
+        <div class="ibexa-modal__trash-option">
+            <p class="ibexa-modal__option-label ibexa-label form-label required">
```

Comment:
label with "*" as an explanation is probably not the best UX.

Reply:
@dew326 Don't we have the same for Conflicts with relations and Sub-items warnings?

Reply:
Not in this styling. The explanation is bigger and not gray.

Reply:
Ok, should I go with a simple paragraph then? Or any other example which html elements/classes should be used?

Reply:
Maybe we should add a new "trash_option" for example: "Drafts" and there put this explanation.
This is probably for @NataliaBecla and @juskora.

Reply:
Hi @NataliaBecla, can we get your feedback on this?

Reply:
@barw4 It seems there was a misunderstanding, as I thought we had resolved everything during the meeting. I suggest we go with @dew326 suggestion to use "Draft" as the title and format the sentence "sending this content item..." in regular font as shown below.

Reply:
okay, will do, thank you

Reply:
Great @barw4 👌👍 Looks good